lang/da.po

```diff
@@ -0,0 +1,96 @@
+msgid ""
+msgstr ""
+"Language: da\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# Et sted til personlig anerkendelse!
+# Skriv noget lignende "Dansk oversættelse af Jvlivs Gaivs\n" her.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "--\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d fri pakke\n"
+msgstr[1] "%d frie pakker\n"
+
+msgid "NONFREE_PACKAGES_COUNT\n"
+msgid_plural "NONFREE_PACKAGES_COUNT\n"
+msgstr[0] "%d ikke-fri pakke\n"
+msgstr[1] "%d ikke-frie pakker\n"
+
+msgid "RMS_HAPPY\n"
+msgstr "Kun frie pakker - rms er stolt!\n"
+
+msgid "RMS_DISAPPOINTED\n"
+msgstr "Over 10%% ikke-fri software-pakker. Kan du ikke lide frihed?\n"
+
+msgid "HELP_USAGE\n"
+msgstr "Usage: vrms-rpm [valgmuligheder]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Vis rms ASCII-kunst når ingen ikke-fri pakker er fundet,\n"
+       "    eller når ikke-fri pakker er 10%% eller mere af det totale."
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Styrrer om terminal escape sekvens skal bruges\n"
+       "    til at farve output. Standardinstillingen er 'auto', som bruger farvet output\n"
```

Comment:
The lines for this message should wrap at 80 characters. You're free to add more linebreaks (`\n`) if you need them.